RoboFile.php

```diff
@@ -613,6 +613,9 @@ public function releaseCreatePullRequest($version) {
   public function releasePublish($version = null) {
     $version = $this->releaseVersionGetNext($version);
     return $this->collectionBuilder()
+      ->addCode(function ()  use ($version) {
```

Comment:
Extra space in there. Unfortunately this seems to be one of the files we don't sniff...